elements/pfe-jump-links/src/pfe-jump-links.js

```diff
@@ -312,11 +330,22 @@ class PfeJumpLinksPanel extends PFElement {
 
   connectedCallback() {
     super.connectedCallback();
+
+    this.scrollTarget =
+      this.getAttribute("pfe-c-scrolltarget") ||
+      this.getAttribute("scrolltarget");
+
     this.nav = this._getNav();
+
     this._init();
-    this.sectionMargin = this.getAttribute("pfe-c-offset");
+
+    this.sectionMargin =
+      this.getAttribute("pfe-c-offset") || this.getAttribute("offset");
     this.customVar = this.cssVariable("--pfe-jump-links-panel--offset") || 200;
-    if (this.nav && this.nav.hasAttribute("pfe-c-autobuild")) {
+    if (
+      (this.nav && this.nav.hasAttribute("pfe-c-autobuild")) ||
+      this.nav.hasAttribute("autobuild")
+    ) {
```

Comment:
The `hasAttribute` call for `autobuild` should also be run only when `this.nav` is true.

```suggestion
    if (
      this.nav && (this.nav.hasAttribute("pfe-c-autobuild") || this.nav.hasAttribute("autobuild"))
    ) {
```